datamodel/changelogs/2025.0.2/02_extensions_agxx_od.py

```diff
@@ -0,0 +1,83 @@
+try:
+    import psycopg
+except ImportError:
+    import psycopg2 as psycopg
+
+from pum.core.deltapy import DeltaPy
```

Comment:
In pum 1.x there is no more python deltas, they need to be pure SQL (with parameters)